apps/client/src/app/components/admin-market-data/asset-profile-dialog/asset-profile-dialog.component.ts

```diff
@@ -152,6 +152,17 @@ export class AssetProfileDialog implements OnDestroy, OnInit {
       });
   }
 
+  public onUnsetBenchmark({ dataSource, symbol }: UniqueAsset) {
+    this.dataService
+      .deleteBenchmark({ dataSource, symbol })
+      .pipe(takeUntil(this.unsubscribeSubject))
+      .subscribe(() => {
```

Comment:
I think the reload is disturbing and not needed here. What about:

```
.subscribe();
```

Reply:
Resolved in latest commit.

Instead added
` this.isBenchmark = false;`